src/main/java/com/dungeoncrawler/Javiarenka/character/Skill.java

```diff
@@ -0,0 +1,58 @@
+package com.dungeoncrawler.Javiarenka.character;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class Skill {
+    @Getter
+    @Setter
+    private String name;
+    @Getter
+    @Setter
+    private int additionalDamage;
+    @Getter
+    @Setter
+    private int additionalHp;
+    @Getter
+    @Setter
+    private boolean isMagicSkill;
+    @Getter
+    @Setter
+    private boolean isPhysicalSkill;
+    @Getter
+    @Setter
+    private SkillTarget target;
+    @Getter
+    @Setter
+    private int coolDown;
+    @Getter
+    @Setter
+    private int manaCost;
+    @Getter
+    @Setter
+    private int staminaCost;
+    @Getter
+    @Setter
+    private CharacterStatus appliesStatus;
+    @Getter
+    @Setter
+    private int appliedStatusDuration;
```

Comment:
O proszę, nice. Ja już gdzieś też zrobiłem pierwsze transfery do Lomboka.

Reply:
no, fajne to bo klasa wygląda dużo przejrzyściej